### DISCUSSION TITLE
Welcome to OMDB-Back-End Discussions!

### DISCUSSION THREAD

**Discussion by cerberusmaster** [cerberusmaster/OMDB-Back-End#3] [Q&A] [answered]
### Discussed in https://github.com/cerberusmaster/OMDB-Back-End/discussions/1

<div type='discussions-op-text'>

<sup>Originally posted by **cerberusmaster** November 30, 2022</sup>

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

</div>

**Comment by cerberusmaster** [comment]
Hello

how can you help me.
I get from `Cannot catch QueryException in Laravel`

**Comment by FFS28** [comment, marked as answer]
You'll set database in .env file.
And then `php artisan migrate:fresh --seed`

**Comment by cerberusmaster** [comment]
Thanks. Greate.
It works well
